src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // перетворити рядок на масив де елемент це правило css
+  // перевірити чи рядок не порожній
+
+  const listStyles = sourceString
+    .split(';')
+    .filter((string) => string.trim() !== '');
+
+  const stylesObject = {};
+
+  listStyles.forEach((entry) => {
+    const [key, value] = entry.split(':').map((el) => el.trim());
+
+    if (key && value) {
+      stylesObject[key] = value;
+    }
+  });
+
+  return stylesObject;
```

Comment:
The function `convertToObject` is well structured and follows the checklist by using array iteration methods instead of loops. However, the variable naming `listStyles` could be improved to reflect its contents more clearly, such as `styleEntries` or `styleRules`.